src/index.html

```diff
@@ -5,8 +5,385 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__element">
+            <a class="nav__link is-active" href="#home">Apple</a>
+          </li>
+
+          <li class="nav__element">
+            <a class="nav__link" href="#home">Samsung</a>
+          </li>
+
+          <li class="nav__element">
+            <a class="nav__link" href="#home">Smartphones</a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#home"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a class="nav__link" href="#home">Gadgets</a>
+          </li>
+
+          <li class="nav__element">
+            <a class="nav__link" href="#home">Tablets</a>
+          </li>
+
+          <li class="nav__element">
+            <a class="nav__link" href="#home">Photo</a>
+          </li>
+
+          <li class="nav__element">
+            <a class="nav__link" href="#home">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__card">
```

Comment:
Better use another class name for this element. Will we have secondary_card? You can call, for example, main_cards or something. 
It should be specific